### ISSUE TITLE
troll reduction applies a -1 armor penlaty

### ISSUE THREAD

**Issue by bulldogc** [chummer5a/chummer5a#5002]
Adding troll reduction on troll removes 1 armor in addition to raising charisma by 1 per rating.

The fluff briefly mentions "dermal-plate removal" but there is nothing that mentions the armor loss in the actual effect of the reduction. Beyond that even if it does remove dermal plates, the reduction applies this penalty even to troll types that don't get dermal plating such as minotaurs.

**Comment by DelnarErsike** [comment]
Marked it as needing clarification because I would assume that removing the dermal plating also removes the armor granted by said dermal plating, but I understand that Chrome Flesh does not explicitly say this happens and so it needs clarification.

**Comment by bulldogc** [comment]
Agreed it is a bit unclear. It def shouldn't be applying the penalty to troll variants that don't actually get dermal armor though. Can't shave down plating that isn't there.
Fomorian, Minotaur, Cyclops

**Comment by chummer5a** [comment]
Ruling was originally taken from the core book reference that installation of orthoskin removes the dermal armour bonus but yeah, it's squirrelly. We probably need to implement something that would add a floor to the reduction.

**Comment by bulldogc** [comment]
Was there a reason the troll bonus armor is being applied as an invisible armor mod instead of adding a free quality for troll armor? If there was a quality for it like infravision ext the cyber could just diable the quality. That way if the quality isnt there it doesn't get "removed"
